nomination/templates/nomination/url_add.html

```diff
@@ -238,6 +238,7 @@ <h4 class="panel-title">Add a Site/URL</h4>
                 initForm();
                 focusAddForm();
                 bindPreviewURL();
+                new SlimSelect({select: '.multi-select'});
```

Comment:
In cases where the nomination project doesn't have a multi-select field in the nomination form, we will still be using this...not sure if it is something to care about or how easily it could be left out when not relevant for the project...

Reply:
The thought did occur to me, but I felt like it didn't really warrant adding iteration and tests to leave out when not used. Also, if we _do_ use slim select for single selections too, then there seems even less reason to go that route since it would be _less_ likely and add just one more part to the check, but that's just my opinion :smile:

Reply:
So I've made the requested change not only here for the instantiation of the selection boxes, but I've also moved the CDN out of the base template and into the two templates where it's actually used, and even in those slim select is only retrieved when needed.